examples/scenes/Cargo.toml

```diff
@@ -23,3 +23,6 @@ instant = "0.1"
 byte-unit = "4.0.19"
 inquire = "0.7"
 ureq = "2.9.6"
+
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+getrandom = { version = "0.2.15", features = ["js"] }
```

Comment:
We should probably have a comment here, because we don't use this dependency ourselves. I recognise that the corresponding Vello example almost certainly doesn't have such a comment.

Reply:
Are we using getrandom currently? I don't think the "example" code (e.g. `with_winit`) has been updated in a long time. I am assuming this was copy/pasted from vello which does use it?

Reply:
@simbleau `rand` requires a source of randomness. To get this, it uses the `getrandom` crate, but for WebAssembly, it doesn't enable anything by default for `wasm32-unknown-unknown` targets, so we have to tell it to enable something:

https://github.com/rust-random/getrandom?tab=readme-ov-file#webassembly-support

(The config here should be more specific and only happen for `wasm32-unknown-unknown` but that's a separate thing.)

Reply:
I've ran into that issue many times and am aware, but I don't see any new use of rand, or any changes to the example, making me wonder why it's being added now?

Reply:
Without looking, I assume because updating the CI builds more now for wasm32 and so this is related to that. But you can probably verify that.

Reply:
It was always required, the old CI just wasn't good enough to catch the issue.